app/controllers/tags_controller.rb

```diff
@@ -1,5 +1,14 @@
 class TagsController < ApplicationController
+
+  def index
```

Comment:
Why is there a conditional here? The index page will list all of the tags as links to their show page.